packages/manager/modules/pci/src/projects/project/public-gateways/public-gateways.controller.js

```diff
@@ -21,4 +21,12 @@ export default class PublicGatewaysController {
   refreshMessages() {
     this.messages = this.messageHandler.getMessages();
   }
+
+  onAddPublicGatewayClick() {
+    this.trackPublicGateways('add');
+  }
```

Comment:
No action to add a **Public Gateway** ?

Reply:
@mohammed-zahaf  it was there but I removed after Anoop's comment, as currently add/config page is in block state.